electron/main/Prompts/Prompts.ts

```diff
@@ -6,13 +6,12 @@ export interface PromptWithContextLimit {
 
 export function createPromptWithContextLimitFromContent(
   content: string | DBEntry[],
+  basePrompt: string,
   query: string,
   tokenize: (text: string) => number[],
   contextLimit: number
 ): PromptWithContextLimit {
-  const basePrompt =
-    "Answer the question below based on the following notes:\n ";
-  const queryPrompt = `Question: ${query}`;
+  const queryPrompt = `Given the above: ${query}`;
```

Comment:
What is the point of having this line? Couldn't we just manage this prompt alongside the basePrompt argument in the caller?